Extensions/probing.hh

```diff
@@ -259,9 +259,8 @@ inline double CalculatePseudoEnergy(double data, std::string modifier, double sl
 
 }
 //END STOLEN FROM RNASTRUCTURE
-
-inline double getReactivityScore(const Subsequence &leftBase, const bool isUnpaired) {
-	static bool isLoaded = false;
+inline double getReactivityScore(const Subsequence &leftBase, const bool isUnpaired, const unsigned int offset) {
```

Comment:
we should rename `leftBase` into something more appropriate like `parsedTerminal` or `inputSubseq`